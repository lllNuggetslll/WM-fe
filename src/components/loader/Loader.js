import React from "react";
import ReactLoader from "react-loader";

const Loader = ({ isLoaded }) => {
  return (
    <ReactLoader
      loaded={isLoaded}
      lines={13}
      length={20}
      width={10}
      radius={30}
      corners={1}
      rotate={0}
      direction={1}
      color="#00CDBD"
      speed={1}
      trail={60}
      shadow={false}
      hwaccel={false}
      zIndex={2e9}
      top="50%"
      left="50%"
      scale={1.0}
      loadedClassName="loadedContent"
    />
  );
};

export default Loader;
