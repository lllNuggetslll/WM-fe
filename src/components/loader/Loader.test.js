import React from "react";
import { shallow } from "enzyme";
import Loader from "./Loader";

describe("Loader", () => {
  it("should render", () => {
    const isLoaded = false;
    const wrapper = shallow(<Loader {...{ isLoaded }} />);

    expect(wrapper).toMatchSnapshot();
  });
});
