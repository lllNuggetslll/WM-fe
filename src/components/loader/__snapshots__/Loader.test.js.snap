// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render 1`] = `
<Component
  color="#00CDBD"
  component="div"
  corners={1}
  direction={1}
  hwaccel={false}
  left="50%"
  length={20}
  lines={13}
  loaded={false}
  loadedClassName="loadedContent"
  parentClassName="loader"
  radius={30}
  rotate={0}
  scale={1}
  shadow={false}
  speed={1}
  top="50%"
  trail={60}
  width={10}
  zIndex={2000000000}
/>
`;
